src/utils/ProofOfWork.ts

```diff
@@ -0,0 +1,18 @@
+// Code copied from https://github.com/decentraland/pow-authorization-server
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const crypto = require('crypto')
```

Comment:
I think that if it is possible, we should use `import {createHash,randomBytes} from 'crypto'` to add type safety